app/views/about/_version.html.haml

```diff
@@ -0,0 +1,4 @@
+.panel
+  .panel-header= t 'about.version'
+  .panel-body
+    %strong= Mastodon::VERSION
```

Comment:
Minor point, but as a general rule I try to avoid using constants directly in views ... can you wrap this up in InstancePresenter (view it to see how other values are already doing this), and then pass that in to the partial (see how `contact` is already doing that)?